src/data/markdown/docs/40 xk6-disruptor/01 Get started/02 Requirements.md

```diff
@@ -3,26 +3,28 @@ title: 'Requirements'
 excerpt: 'Requirements for using xk6-disruptor in your test scripts'
 ---
 
-The xk6-disruptor is a k6 extension. 
-To use it in a k6 test script, you need to bundle a k6 extension that includes the disruptor.
-Refer to the [Installation](/javascript-api/xk6-disruptor/get-started/installation) section  for instructions on how to get this custom build.
+At this moment,
+xk6-disruptor is a k6 extension **intended for systems running Kubernetes version 1.23 or higher**.
 
- xk6-disruptor needs to interact with the Kubernetes cluster on which the application under test is running.
- To do so, you must have the credentials to access the cluster in a [kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) file.
- Ensure that this file is pointed to by the `KUBECONFIG` environment variable or that it is located at the default location, `$HOME/.kube/config`.
 
-<Blockquote mod="note">
+To use xk6-disruptor in a k6 test script, you need the following:
 
-xk6-disruptor requires Kubernetes version 1.23 or higher
+- **A bundled k6 extension that includes the disruptor.**
```

Comment:
@MattDodsonEnglish I find the term "a boundle k6 extension" confusing. Is that the way we are referring now to the custom k6 binaries with bundled extensions?

I think the original phrase was something like "a custom k6 binary that bundles the xk6-disruptor extension".